subsys/bluetooth/host/adv.c

```diff
@@ -49,13 +49,23 @@ struct ad_stream {
 	 * The length and type are included in this offset.
 	 */
 	uint16_t current_ltv_offset;
+
+	/* The remaining size of total ad[i].data[j].data_len + 2 for LTV header */
+	size_t remaining_size;
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/43255/files#r817569322

Not and modified?